mezzanine/pages/static/mezzanine/js/admin/page_tree.js

```diff
@@ -59,21 +59,13 @@ jQuery(function($) {
         });
     }
 
-    // The dropdown list for adding a new page contains URLs for each
-    // model - redirect when selected.
     $('.addlist').change(function() {
         // Ensure the branch is saved as open when adding to it so that
         // the new branch will be visible directly after saving.
         var id = $(this).attr('id');
         if (id) {
             toggleID(true, id.split('-')[1]);
         }
-        var addUrl = this[this.selectedIndex].value;
-        if (addUrl) {
-            location.href = addUrl;
-        }
-        this.selectedIndex = 0;
```

Comment:
The purpose of the conditional `if (addUrl)` and `this.selectedIndex = 0` is unclear to me. The only `<option>` that doesn't have a `value` is the first one (`<option value="">{% trans "Add" %} ...</option>`) which is always selected on page load so it cannot trigger the `change` handler.